angular/src/components/login.component.ts

```diff
@@ -61,6 +65,20 @@ export class LoginComponent implements OnInit {
         if (Utils.isBrowser && !Utils.isNode) {
             this.focusInput();
         }
+
+        let webVaultUrl = this.environmentService.getWebVaultUrl();
+        if (webVaultUrl == null) {
+            webVaultUrl = 'https://vault.bitwarden.com';
```

Comment:
I really dislike these defaults. While probably out of scope for this change, It's my opinion we should open a ticket to move those to the `environmentService` to keep things consistent.

Reply:
I've thought that before too, At the time when I looked into it there was a reason, but I can't remember it right now.

I think it was something along the lines of needing to override null values for development environments.

Reply:
Ah yes, it's for Send: https://github.com/bitwarden/jslib/blob/master/angular/src/components/send/send.component.ts#L174-L177

Could just make a new helper for that too https://app.asana.com/0/1169444489336079/1200631805316488/f